frontend/src/Views/Export/ExportPage.tsx

```diff
@@ -1,14 +1,46 @@
 /**
- * Hello World page. Nothing Special. This is a skeleton!!!
- * @desc only shows Hello World, will implement as Designer creates mockup
- * @author @Kcaparas
+ * @desc Export page for downloading your work in nmp format
+ * @author @GDamaso
  */
 
-import React from 'react';
+import Button from '@Commons/Button/Button.tsx';
+import MainPageHeader from '@Commons/MainPageHeader/MainPageHeader.tsx';
+import MainPageFooter from '@Commons/MainPageFooter/MainPageFooter.tsx';
+import { FC } from 'react';
+import { StyledContent, StyledLandingContainer } from './ExportPage.styles.ts';
 
-const ExportPage: React.FC = () => {
-  localStorage.clear();
-  return <h2>Export Page</h2>;
+const ExportPage: FC = () => {
+  const downloadFile = () => {
+    localStorage.clear();
+    console.log('Downloading');
+  };
+
+  return (
+    <StyledLandingContainer>
+      <MainPageHeader />
+      <StyledContent>
+        <div id="dataFileHeader">
+          <h6>NMP Data File </h6>
+        </div>
+        <p>To continue later, Download this file to your computer or mobile device.</p>
+        <p>Load a NMP file on the Homepage when you want to continue.</p>
+        <a href="https://nmp.apps.nrs.gov.bc.ca/Report/Report#">
```

Comment:
This link doesn't work. If you link here the real nmp has no context of any work done so it shows error messages
Also by clicking it you make my tab change to nmp, so I lose context of my work in the React app. 
You could make this open in a new tab with `target="_blank"`

Reply:
Also should've mentioned, your `p` tag and `a` tags are backwards, an `p` tag wraps the `a` tag, not the other way around. a screen reader or even your inspect tools will pick that link up at a `p` tag incorrectly

Reply:
Fixed in PR:
https://github.com/bcgov/nr-sustainment-capstone-2024/pull/198

Created a CustomLink to be used in place of ButtonLinks :smile: